packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -361,7 +361,7 @@ function UntranslatedComplexItem(props) {
 						key="badge"
 						className={`${theme.badge} badge`}
 						aria-label={t('TC_OBJECT_VIEWER_NB_CHILD', {
-							defaultValue: 'Contains {{count}} child',
+							defaultValue: 'Contains {{count}} child object',
```

Comment:
`objects` ?

Reply:
Same as above, there is a separate entry for plural : TC_OBJECT_VIEWER_NB_CHILD_plural

Reply:
defaultValue_plural